user/build-stages.md

```diff
@@ -97,6 +97,25 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+### Naming your Jobs within Build Stages
+
+You can also name specific jobs within build stages. We recommend unique job names, but 
+do not enforce it (though this may change in the future). Job names are defined like other 
+job configuration elements within a build stage, but the value of the `name` must be in quotes.
+Here's an example of a build using build stages and job names:
+
+```yaml
+  include:
+    - stage: test
+      name: "test 1"
```

Comment:
Shall we call this Test Job 1.... for extra clarity?

Reply:
How about `Unit tests` and `Integration tests`, as more meaningful example names?

Reply:
thanks! yeah - I gave them some better names and changed the section from `Test` to `Tests` to show, well, multiple tests happen here